### ISSUE TITLE
www.imvertor.org geeft een 404

### ISSUE THREAD

**Issue by Bakkej** [Imvertor/Imvertor-Maven#470]
De URL (www.imvertor.org) die gekoppeld is aan deze repository geeft een 404. 

![image](https://github.com/Imvertor/Imvertor-Maven/assets/101263024/1acf42f9-87e7-47e7-ae38-fd5d12ee26dc)

**Comment by ArjanLoeffen** [comment]
Ja, moeten we nog naar kijken. Het is altijd onduidelijk geweest wat daar moet komen en van wie deze site is. Imvertor is formeel een OS project gedoneerd door Kadaster. 

Dit is een interessante vraag voor de UG.
